crates/vhost-device-gpio/src/backend.rs

```diff
@@ -287,14 +370,26 @@ mod tests {
         assert_eq!(config, expected_config);
     }
 
+    #[cfg(not(feature = "mock_gpio"))]
```

Comment:
You can do a `mod mock_gpio_tests` and put the `cfg` check onto that to avoid needing to repeat it everywhere.